controller/login.go

```diff
@@ -145,10 +145,10 @@ func (c *LoginController) Refresh(ctx *app.RefreshLoginContext) error {
 	case 400:
 		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError(res.Status+" "+rest.ReadBody(res.Body)))
 	default:
-		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(errs.New(res.Status+" "+rest.ReadBody(res.Body))))
+		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, errs.New(res.Status+" "+rest.ReadBody(res.Body))))
```

Comment:
maybe we could add a text here `unknown error when refreshing the keycloak token`